### DISCUSSION TITLE
IDH-Alarms - exempt IP-addresses of vulnerability scanners from generating false positives

### DISCUSSION THREAD

**Discussion by ejgh-oe** [Security-Onion-Solutions/securityonion#12547] [Unsupported Versions] [answered]
Hi,
Our vulnerability scanners are generating a lot of false positives when hitting our internal honeypots (SO IDH-Nodes). 
What's the best way to prevent these alarms from being generated in the first place?

I was thinking about a BPF, but where should I put the BPF? BPF for PCAP, BPF for Zeek or BPF for Suricata? 
IDH-alarms come from playbook, but in the end it boils down to there the packets come from.

Thanks much in advance for any clue.

**Comment by InfosecGoon** [comment]
Sorry, just to be clear -- the alerts are being generated by Playbook (your vulnerability scanners interacting with the honeypot services) or by Suricata (the traffic from your scanners to the honeypot node)?

**Reply by ejgh-oe** [reply]
The IDH alerts are generated by playbook - e.g.
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/84927703/3f108b22-f02b-4bea-9419-c38727659517)

and in detail view

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/84927703/e530cd5f-f0dd-4191-9fd3-b0a94d2f0414)

The corresponding playbook definition (Play #2134) reads 

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/84927703/e70e4e30-0228-4a0d-9c10-05646d200008)


As pointed out in my original question I'd like to suppress these alerts for certain source-IP (vulnerability-scanners) while leaving it intact (i.a. alarming) for all others.

**Reply by izidood** [reply]
just some additional info, you can make an ip ignore list in idh advanced tap. then only the idh's will ignore the scanners.

**Reply by ejgh-oe** [reply]
> just some additional info, you can make an ip ignore list in idh advanced tap. then only the idh's will ignore the scanners.

The corresponding dialog box only says "it's YAML" and "be careful" ;-)

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/84927703/d9783255-be59-48e6-bcb4-050cf11eab77)

Any ideas about the syntax? I haven't found anything in the docs, i.e. https://docs.securityonion.net/en/2.4/idh.html#configuration.

**Reply by izidood** [reply, marked as answer]
so i havn't tested it myself, i will in a bit.
i just put in the ignore list in the default.yaml file for IDH, which is fine, but i think if you upgrade node it wont save it.
im thinking it might have to  look like this perhaps ?
```yaml
idh:
  opencanary:
    config:
      ip.ignorelist: [x.x.x.x.x] 
```

i'll test it out myself as well at some point.

**Reply by jaypfe** [reply]
@gitgoodgod 

Your solution is tested functional for me!

Not OP but thanks for the input on this thread, really saved me some headache.

**Reply by izidood** [reply]
awesome m8, seems like it also worked for me.

**Comment by ejgh-oe** [comment]
Thanks @gitgoodgod - definitely helps getting out those false alarms triggered by vulnerability scans :-)

**Reply by izidood** [reply]
great!
